### DISCUSSION TITLE
Sort Journey

### DISCUSSION THREAD

**Discussion by markushartmann-dev** [bassmaster187/TeslaLogger#1204] [Q&A]
Hi there,
first of all, i love the TeslaLogger, thanks for providing such a great tool.

Is it possible or maybe in the near future, when adding a new journey, to sort the start date by the newest entries first? Because i use this tool for reporting and billing, have currently around 50 Journeys in it and when creating a new Journey, i have to scroll down many pages with something around 15 months of data...

screenshot for example attached:
![image](https://github.com/bassmaster187/TeslaLogger/assets/92719027/03bfc61f-2143-460b-a967-3fbbfe2dbec4)

**Comment by superfloh247** [comment]
My first implementation had a dropdown with search functionality, maybe we can restore that. 

use https://select2.org/getting-started/basic-usage

implemented in Journeys.cs here https://github.com/bassmaster187/TeslaLogger/blob/36d40186f135c7728d322858fe9159690fdc58a3/TeslaLogger/Journeys.cs#L69

**Reply by markushartmann-dev** [reply]
this would be nice <3

**Comment by superfloh247** [comment]
I don't touch php

**Comment by bassmaster187** [comment]
fixed with https://github.com/bassmaster187/TeslaLogger/commit/4654475a1344189ddc6372b92f93ffb441919f04

![image](https://github.com/bassmaster187/TeslaLogger/assets/6816385/076e6376-5154-4d60-a79e-2a005bf7e6b4)

**Comment by markushartmann-dev** [comment]
vielen lieben Dank / Thank you so much 👍
